elasticsearch_dsl/document.py

```diff
@@ -150,6 +151,32 @@ def get(cls, id, using=None, index=None, **kwargs):
         return cls.from_es(doc)
 
     @classmethod
+    def mget(cls, ids, using=None, index=None, missing='none', **kwargs):
+        if missing not in frozenset(('raise', 'skip', 'none')):
+            raise ValueError("'missing' must be 'raise', 'skip', or 'none'.")
+        es = connections.get_connection(using or cls._doc_type.using)
+        index = index or cls._doc_type.index
+        docs = es.mget(
+            {'ids': ids},
+            index=index,
+            doc_type=cls._doc_type.name,
+            **kwargs
+        )
+        objs, missing_ids = [], []
+        for doc in docs['docs']:
+            if doc['found']:
+                objs.append(cls.from_es(doc))
+            else:
+                if missing == 'none':
+                    objs.append(None)
+                elif missing == 'raise':
+                    missing_ids.append(doc['_id'])
+        if missing == 'raise' and missing_ids:
+            message = 'Documents {} not found.'.format(', '.join(missing_ids))
+            raise NotFoundError(404, message)
```

Comment:
I would love to see the actual missing docs to be added as the third parameter - a list of the missing records, not just the IDs - do `missing_ids.append(doc)` and raise `NotFoundError(404, message, missing_ids)`